chrome-launcher/chrome-launcher.ts

```diff
@@ -216,7 +230,7 @@ export class Launcher {
   }
 
   // resolves when debugger is ready, rejects after 10 polls
-  private waitUntilReady() {
+  waitUntilReady() {
```

Comment:
I exposed this for testability... I theoretically i could mock net.createConnection and emit a fake event here... instead